example/ts/__relay_artifacts__/AddTodoMutation.graphql.ts

```diff
@@ -1,10 +1,7 @@
 /* tslint:disable */
 
 import { ConcreteRequest } from "relay-runtime";
-export type AddTodoInput = {
-    readonly text: string;
-    readonly clientMutationId?: string | null;
-};
+export type AddTodoInput = {};
```

Comment:
These changes shouldn’t be happening. Not sure yet why it does.

/cc @koddsson

Reply:
Hmm, this is weird. My app is not doing this but I am using relay 2.0.

I'm off on holiday from work but I could see if I can poke around in this PR if I have some time.

Reply:
I’ll give Relay 2 a try tomorrow 👍

Reply:
I tried debugging this a bit and I found out that this seems to be some issue with some relay part. Not sure what, but the types are passed to the `formatGeneratedModule.js` function empty so it doesn't seem to be any thing we introduced but some sort of dependency mismatch or even a bug on some version of relay we have.

It's really hard to debug the example code when it's referenced via a packed module, I think if I'm going to debug this anymore I'll remove that resolution thing and just require the `lib/` folder of the parent directory so I can edit the node modules of the package with debug `console.log`s to help me debug it. That will also make sure I don't have to build and pack every time I want to change anything.